### ISSUE TITLE
Issue with the filter

### ISSUE THREAD

**Issue by oofy123** [meower-media-co/Meower-Server#121]

**Comment by MikeDev101** [comment]
This is a bug with Better Profanity.

**Comment by JoshAtticus** [comment]
> This is a bug with Better Profanity.

Not a bug, it's in their readme

**Comment by showierdata9978** [comment]
> > This is a bug with Better Profanity.
> 
> Not a bug, it's in their readme

no, its a bug in Meower-Server because i partly fixed those iirc

**Comment by MikeDev101** [comment]
De-prioritizing issue, and making this a CL4 branch issue.

**Comment by oofy123** [comment]
> De-prioritizing issue, and making this a CL4 branch issue.

Ok
